schema/metrics/histogram_metrics.mapping

```diff
@@ -0,0 +1,159 @@
+{
+  "index_patterns": [
+    "sso_metrics-*-*"
+  ],
+  "dynamic_templates": [
+    {
+      "attributes_map": {
+        "mapping": {
+          "type":"keyword"
+        },
+        "path_match":"attributes.*"
+      }
+    },
+    {
+      "resources_map": {
+        "mapping": {
+          "type": "keyword"
+        },
+        "path_match": "resources.*"
+      }
+    }
+  ],
+  "mappings": {
+    "_meta": {
+      "version": "0.1.0-dev"
+    },
+    "properties": {
+      "temporality": {
+        "type": "text",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "bucketCounts": {
+        "type": "long"
+      },
+      "bucketCountsList": {
```

Comment:
Is it not possible represent the fact that this is a list, here?

Reply:
Unfortunately no see https://opensearch.org/docs/2.0/opensearch/supported-field-types/index/#arrays

Reply:
_"There is no dedicated array field type in OpenSearch. Instead, you can pass an array of values into any field. All values in the array must have the same field type"_